src/tests/homepagecounter.test.js

```diff
@@ -0,0 +1,44 @@
+import homePageCount from '../modules/homePageCounter.js';
+
+// const home = document.createElement("ul");
+// home.classList.add("movie-section");
+// for (let i = 1; i < 7; i += 1) {
+//   const div = document.createElement("div");
+//   home.appendChild(div);
+// }
+
+// describe([home, "(6)"])(
+//   "check number of shows in a page: ",
+//   (mainPage, expected) => {
+//     document.body.innerHTML = `<span class="show-count"></span>`;
+//     document.body.appendChild(mainPage);
+//     const count = document.body.querySelector(".show-count");
+//     homePageCount(count);
+//     test(`test with ${mainPage.childElementCount} number of items in the page:`, () => {
+//       expect(count.innerHTML).toBe(expected);
+//     });
+//   }
+// );
```

Comment:
- [ ] It is not the best practices to leave commented line of code or block of code in your files as this practice makes your codebase unkempt. I would recommend you to remove them in order to make your codebase clean.

Reply:
Thank you for this feedback, I will delete these commented codes.